README.md

```diff
@@ -1,7 +1,7 @@
-# nRF Connect SDK Example Application
+# nRF Connect SDK Example application
```

Comment:
why keep the upper case in "Example" then?

Reply:
So true. I was first thinking of it as a name for the application, thus capitalized Example. Now, thinking again and I think you are right. Let's have it all lower case.